data/websites.json

```diff
@@ -770,6 +770,11 @@
       "url": "http://nationaldata.gov.np/",
       "description": "National Data Portal Nepal"
     },
+    {
+      "name": "Nepal Government",
```

Comment:
Can you please remove this entry?
The website is already on the list.

Reply:
> Can you please remove this entry? The website is already on the list.
I re-checked the existence of that website(https://nepal.gov.np/) but I didn't find another one. Can you please show me where is the other one?